### DISCUSSION TITLE
Hero 12 Labs firmware v2.10.70 no LEVL

### DISCUSSION THREAD

**Discussion by SmetcH1302** [gopro/labs#773] [Q&A]
With newest software on-screen spirit disaperead and after scaning generated QR code for LEVL nothing happens!?

**Comment by dnewman-gpsw** [comment]
This have been reported https://github.com/gopro/labs/issues/768  with a workaround.

But for unrelated reasons this build includes a patch:
https://www.dropbox.com/scl/fi/jsfxxywryr36r4wihu1ni/LABS_HERO12_02_10_72.zip?rlkey=70y4mt3l6r9b2k5aypu3kvi3w&dl=0

The official Labs release with this fixed will be in a month or two.

**Comment by JiDey0007** [comment]
I didn't quite understand what is the problem with the last update only on the 12 and the benefit / risk

**Reply by dnewman-gpsw** [reply]
I don't under your question.  If you want LEVL working, use it.

**Reply by JiDey0007** [reply]
I use GoPro laps for fast parameter migration in a box during diving and to have maximum writing performance especially in underwater and underwater environments with difficult contrast

But I'm not an expert and I don't know what LEVL is? ( Exhibition? ) and what's the problem with the update, I'm French google translated as a first level schoolboy 😅

To be clear, what does it change the February update?

**Reply by dnewman-gpsw** [reply]
Best not to follow up on someone else's question, as I will assume the post is strongly related.  LEVL is explained with the other extensions https://gopro.github.io/labs/control/extensions/  

Stick with the release regular Labs releases, as they are more tested. https://gopro.github.io/labs/

**Reply by JiDey0007** [reply]
Ok, I'm not using LEVL, so I'm going to do the February update on my 12
